mcstatus/status_response.py

```diff
@@ -91,7 +91,7 @@ def description(self) -> str:
 
     @classmethod
     @abstractmethod
-    def build(cls, *args, **kwargs) -> Self:
+    def build(cls, *args: Any, **kwargs: Any) -> Self:  # noqa: ANN401  # Any is bad, but this is abstract
```

Comment:
We do not set type annotations for `*args` and `**kwargs`, just revert it.

```suggestion
    def build(cls, *args, **kwargs) -> Self:
```